pkg/kyverno/test/test_command.go

```diff
@@ -863,6 +863,27 @@ func applyPoliciesFromPath(fs billy.Filesystem, policyBytes []byte, isGit bool,
 			}
 		}
 
+		// check request.operation variable present in the policy
+		for _, vars := range matches {
+			vari := strings.Join(vars, " ")
+			if strings.Contains(vari, "request.operation") {
+				if globalValMap["request.operation"] == "" {
+					if globalValMap == nil {
+						globalValMap = make(map[string]string)
+						// set globally, request.operation to CREATE
+						globalValMap["request.operation"] = "CREATE"
+						log.Log.V(3).Info("variable request.operation found in policy, by default globally set to CREATE.")
+						break
+					} else {
+						// set globally, request.operation to CREATE
+						globalValMap["request.operation"] = "CREATE"
+						log.Log.V(3).Info("variable request.operation found in policy, by default globally set to CREATE.")
```

Comment:
@viveksahu26 I don't understand logic here, In both cases why we setting to value to "CREATE"